package.json

```diff
@@ -2,8 +2,13 @@
   "name": "protractor-helper",
   "version": "3.6.13",
   "scripts": {
-    "test": "webdriver-manager update && protractor test/protractor.conf.js"
+    "pretest": "webdriver-manager update",
+    "test": "protractor test/protractor.conf.js",
+    "prettify": "prettier --write **/*.js **/*.md package.json sampleApp/**/*"
```

Comment:
Will files inside `constants_and_utils` directory be prettified as well with this config?

Reply:
Yes, when the `prettify` runs, the following ocurrs:

![image](https://user-images.githubusercontent.com/29241659/48901035-4d2bd700-ee3b-11e8-9f17-6e912699b0ef.png)

The script `prettify` is `prettier --write **/*.js **/*.md package.json sampleApp/**/*`.

 Explaining him, the `**/*.js` command find all javascript files in the same directory and subfolder:
 ```
  constant_and_utils\constants.js
  constant_and_utils\utils.js
  index.js
  test\constants.js
  test\protractor.conf.js
  test\sample.spec.js
 ```

 The same ocurrs with `.md`, finding all markdoowns in the same directory and subfolder.
```
.github\ISSUE_TEMPLATE.md
.github\PULL_REQUEST_TEMPLATE.md
CHANGELOG.md
docs\CONTRIBUTING.md
docs\EXAMPLES.md
README.md
```

The `package.json` is self explanatory.

The `sampleApp/**/*` find all files inside sampleApp folder and all files inside the subfolder on sampleApp.
```
sampleApp\index.html
sampleApp\public\style.css
```
## Explain 2 asterisks
 The follow comand find javascript files only in any subfolder.
`*/*.js`

 The follow comand with 2 asterisks find all javascript files in the same directory and in any subfolder.
`**/*.js`

 That is, the `**/*.js` command is equal the following two commands: `*.js */*.js`.

Reply:
A bit more information about prettier:
 When he format any file, the color com command line change from grey to white.
 Example formatting the package.json file:

![image](https://user-images.githubusercontent.com/29241659/48901716-7483a380-ee3d-11e8-9389-542ad3467584.png)

Reply:
Cool. Thanks!